frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue

```diff
@@ -160,6 +165,8 @@ async function fetchHistoricPrices() {
 }
 
 watch([datetime, asset], async () => {
+  if (get(hidePriceFields))
+    return;
```

Comment:
This is redundant check, isn't it? You also check it inside `fetchHistoricPrices()`

Reply:
also make it part of the watch.
You need to test, by set it first to informational,
and then set it to other.
and vice versa.
make sure the price is always fetching when it's not informational, even if we change the event type.